mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule SaladUI.MixProject do
     [
       app: :salad_ui,
       version: "0.14.1",
-      elixir: "~> 1.14",
```

Comment:
Would it be better to leave the version as it is (1.14) and make `:tw_merge` downgrade Elixir version instead?

Reply:
also I made [PR](https://github.com/bluzky/tw_merge/pull/1) in `:tw_merge` in sync with Phx v1.7.17's Elixir version (`1.14`)

Reply:
Ok, I'll keep this PR focused on the live_view update. Thanks!